Source/WebCore/css/StyleProperties.cpp

```diff
@@ -250,7 +250,6 @@ static constexpr bool canUseShorthandForLonghand(CSSPropertyID shorthandID, CSSP
     case CSSPropertyColumns:
     case CSSPropertyContainer:
     case CSSPropertyFontSynthesis:
-    case CSSPropertyGap:
```

Comment:
Is this part covered by WPT? (this is what collapses row-gap + column-gap into gap)

Reply:
oh suddenly you made me wonder about something, aka what is happening with 
`style="row-gap:normal; column-gap:normal"`
insted of just:
`style="gap:normal"`

It's all good. 

```
<div id="target" style="row-gap:normal; column-gap:normal">food</div>

const target = document.getElementById('target');
target.style.cssText;
target.style.getPropertyValue('gap');
window.getComputedStyle(target).gap;


# STP 190
cssText:          "row-gap: normal; column-gap: normal;"
getPropertyValue: "normal"
getComputedStyle: "normal normal"


# This patch (Minibrowser):
cssText:          "gap: normal;"
getPropertyValue: "normal"
getComputedStyle: "normal"


# Firefox 
cssText:          "gap: normal;"
getPropertyValue: "normal"
getComputedStyle: "normal"


# Chrome 
cssText:          "gap: normal;"
getPropertyValue: "normal"
getComputedStyle: "normal"
```

The tests are just covering the value `gap`:
https://github.com/web-platform-tests/wpt/blob/efd5dd8342/css/css-align/parsing/gap-computed.html

```
test_computed_value("gap", "normal");
test_computed_value("gap", "10px");
test_computed_value("gap", "20%");
test_computed_value("gap", "calc(20% + 10px)");
test_computed_value("gap", "calc(-0.5em + 10px)", "0px");
test_computed_value("gap", "calc(0.5em + 10px)", "30px");
```

Maybe our local version of the css-align tests is too old. I should try to import them. 

but this is only testing the `getComputedStyle()` value

The animation tests are also only testing the `getComputedStyle()` values.

Indeed there are no tests for cssText and getPropertyValue.

Reply:
@nt1m I also wonder if I should reimport css-align

Reply:
https://github.com/web-platform-tests/wpt/issues/45334

Reply:
https://github.com/web-platform-tests/wpt/pull/45335